tests/002-test-reject-disk-with-lvm-signature.sh

```diff
@@ -25,17 +27,33 @@ EOF
   done
 
   # Run container-storage-setup
-  $CSSBIN >> $LOGS 2>&1
-
-  # Css should fail. If it did not, then test failed. This is very crude
-  # check though as css can fail for so many reasons. A more precise check
-  # would be too check for exact error message.
-  [ $? -ne 0 ] && test_status=0
+  $CSSBIN > $tmplog 2>&1
+  rc=$?
+  cat $tmplog >> $LOGS 2>&1
+
+  # Test failed.
+  if [ $rc -ne 0 ]; then
+      if grep --no-messages -q "$errmsg" $tmplog; then
+          test_status=0
+      else
+          echo "ERROR: $testname: $CSSBIN Failed for a reason other then \"$errmsg\"" >> $LOGS
+      fi
+  else
+      echo "ERROR: $testname: $CSSBIN Succeeded. Should have failed since LVM2_member signature exists on devices $devs" >> $LOGS
+  fi
 
-  cleanup $vg_name "$devs"
+  cleanup_pvs $devs
```

Comment:
@rhvgoyal Then we have a much bigger problem than the test :)
Let me come to your desk.